dbt/adapters/databricks/python_models/python_submissions.py

```diff
@@ -162,3 +162,161 @@ def submit(self, compiled_code: str) -> None:
 class ServerlessClusterPythonJobHelper(BaseDatabricksHelper):
     def submit(self, compiled_code: str) -> None:
         self._submit_through_notebook(compiled_code, {})
+
+
+class WorkflowPythonJobHelper(BaseDatabricksHelper):
+
+    @property
+    def default_job_name(self) -> str:
+        return f"{self.database}-{self.schema}-{self.identifier}__dbt"
```

Comment:
Think it might be helpful to put the dbt up front.  That way if you sort in the UI, all of the dbt jobs are together.

Reply:
good point, thanks Ben

Reply:
changed